### ISSUE TITLE
Count Cloud IDE as Local

### ISSUE THREAD

**Issue by crasx** [acquia/drupal-environment-detector#21]
Ran into an issue with a blt project where local.settings.php was not being included. Turned out to be because `isLocalEnv` returns false for Acquia cloud IDE. Should this be true? Happy to throw up a PR here or I can fix it downstream in blt. 

Reference call in BLT: https://github.com/acquia/blt/blob/main/settings/blt.settings.php#L132

**Comment by nikunjkotecha** [comment]
LANDO / IDE - while they are more similar to Acquia Cloud (or Cloud Next) their purpose is to serve as development environment only. Now term local is ambiguous but that is what is used today in BLT and all other places.

I vote for considering both IDE and LANDO as local till we have conclusion and work done to remove the usage of "local" from all places.

We have to use separate plugins to make it work like https://github.com/pavlosdan/blt-ide-local as of now which doesn't make sense to me.

**Comment by pavlosdan** [comment]
IMO both the IDE and Lando should be considered "local" environments by the environment detector. For all intents and purposes they are used for "local" development on their respective environments regardless of their similarities to Acquia Hosting.

**Comment by ba66e77** [comment]
> I vote for considering both IDE and LANDO as local till we have conclusion and work done to remove the usage of "local" from all places.

Heavy plus one

**Comment by danepowell** [comment]
Yeah I don't want to get buried in semantics, but I do care about not breaking people's applications, which is what will happen if we go making IDEs local. Then all the people whose applications broke will open an issue asking us to make it non-local again, and we're back to square one.

I want to see a list of every known reference to `isLocalEnv()`, e.g. in BLT and Acquia CLI but also any third-party packages. Then I want to see what breaks if IDEs are considered local.

If the answer is "nothing breaks", then I'm happy to move forward with making Cloud IDE local with a strongly-worded comment to this effect on `isLocalEnv()`.

If stuff does break, very likely we need to split up `isLocalEnv()` into multiple methods that are more meaningfully and intuitively named than "local", which is a relic of the past in the age of Cloud development.
